packages/grpc-js/src/call-stream.ts

```diff
@@ -245,22 +245,22 @@ export class Http2CallStream extends Duplex implements Call {
       stream.on('response', (headers, flags) => {
         switch (headers[HTTP2_HEADER_STATUS]) {
```

Comment:
This line now has to say `headers[':status']` so that TypeScript can recognize that the value will be a number.

Reply:
Updated